am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/importer/DefaultPermissionIntegrationTest.java

```diff
@@ -96,24 +100,33 @@ void shouldRemoveAllEntriesFromTablesWhenValuesExist() {
         service.truncateDefaultPermissionsForService(serviceName, resourceType);
 
         assertThat(databaseHelper.countDefaultPermissions(resourceDefinition,
-            ROOT_ATTRIBUTE.toString(), ROLE_NAME, Permissions.sumOf(ImmutableSet.of(READ)))).isEqualTo(0);
+            ROOT_ATTRIBUTE.toString(), roleName, Permissions.sumOf(ImmutableSet.of(READ)))).isEqualTo(0);
 
         assertThat(databaseHelper.getResourceAttribute(resourceDefinition,
             ROOT_ATTRIBUTE.toString(), PUBLIC)).isNull();
     }
 
     @Test
     void shouldRemoveEntriesWithResourceNameFromTablesWhenEntriesExist() {
-        service.addRole(ROLE_NAME, RESOURCE, PUBLIC, ROLE_BASED);
+        service.addRole(roleName, RESOURCE, PUBLIC, ROLE_BASED);
         service.addResourceDefinition(resourceDefinition);
-        service.grantDefaultPermission(createDefaultPermissionGrant(resourceDefinition, ImmutableSet.of(READ)));
+        service.grantDefaultPermission(grantDefaultPermissionForResource(resourceDefinition, ImmutableSet.of(READ)));
 
         service.truncateDefaultPermissionsByResourceDefinition(resourceDefinition);
 
         assertThat(databaseHelper.countDefaultPermissions(resourceDefinition,
-            ROOT_ATTRIBUTE.toString(), ROLE_NAME, Permissions.sumOf(ImmutableSet.of(READ)))).isEqualTo(0);
+            ROOT_ATTRIBUTE.toString(), roleName, Permissions.sumOf(ImmutableSet.of(READ)))).isEqualTo(0);
 
         assertThat(databaseHelper.getResourceAttribute(resourceDefinition,
             ROOT_ATTRIBUTE.toString(), PUBLIC)).isNull();
     }
+
+    private DefaultPermissionGrant grantDefaultPermissionForResource(ResourceDefinition resourceDefinition,
```

Comment:
Tests in the class visibly pass role name around with exceptions around calls to `grantDefaultPermissionForResource` method. Be consistent.

Reply:
👍